lib/LaTeXML/Package/textcomp.sty.ltxml

```diff
@@ -15,187 +15,8 @@ use strict;
-  textbullet           => "\x{2022}",
-  textdagger           => "\x{2020}",
-  textdaggerdbl        => "\x{2021}",
-  textopenbullet       => "\x{25E6}",
-  textparagraph        => UTF(0xB6),
-  textperiodcentered   => UTF(0xB7),
-  textpilcrow          => UTF(0xB6),
-  textsection          => UTF(0xA7));
-map { DefPrimitiveI('\\' . $_, undef, $textcomp_footnote{$_},
-    bounded => 1, font => { encoding => 'TS1' }) } keys %textcomp_footnote;
-
-#======================================================================
-# Scientific symbols
-
-our %textcomp_scientific = (
-  textcelsius      => "\x{2103}",
-  textdegree       => UTF(0xB0),
-  textdiv          => UTF(0xF7),
-  textlnot         => UTF(0xAC),
-  textmho          => "\x{2127}",
-  textminus        => "\x{2212}",
-  textmu           => UTF(0xB5),
-  textohm          => "\x{2126}",
-  textordfeminine  => UTF(0xAA),
-  textordmasculine => UTF(0xBA),
-  textpm           => UTF(0xB1),
-  textsurd         => "\x{221A}",
-  texttimes        => UTF(0xD7));
-map { DefPrimitiveI('\\' . $_, undef, $textcomp_scientific{$_},
-    bounded => 1, font => { encoding => 'TS1' }) } keys %textcomp_scientific;
-
-#======================================================================
-# Various
-
-our %textcomp_various = (
-  textacutedbl             => "\x{02DD}",
-  textasciiacute           => UTF(0xB4),
-  textasciibreve           => "\x{02D8}",
-  textasciicaron           => "\x{02C7}",
-  textasciidieresis        => UTF(0xA8),
-  textasciigrave           => UTF(0x60),
-  textasciimacron          => UTF(0xAF),
-  textbigcircle            => "\x{25CB}",
-  textblank                => "\x{2422}",
-  textborn                 => "\x{2605}",
-  textdblhyphen            => "=",            # ?
-  textdblhyphenchar        => "=",            # ?
-  textdied                 => "\x{271D}",
-  textdivorced             => "\x{26AE}",
-  textfractionsolidus      => UTF(0x2F),
-  textgravedbl             => "\x{201F}",
-  textinterrobang          => "\x{203D}",
-  textinterrobangdown      => "\x{2E18}",
-  textleaf                 => "\x{1F342}",    # FALLEN LEAF (close, but plane 1)
-  textmarried              => "\x{26AD}",
-  textmusicalnote          => "\x{266A}",
-  textnumero               => "\x{2116}",
-  textquotesingle          => "'",
-  textquotestraightbase    => "\x{201A}",
-  textquotestraightdblbase => "\x{201E}",
-  textrecipe               => "\x{211E}",
-  textthreequartersemdash  => "\x{2014}",     # whole em dash, actually
-  texttildelow             => "~",            # ??
-  texttwelveudash          => "\x{2014}");    # whole em dash, actually.
-map { DefPrimitiveI('\\' . $_, undef, $textcomp_various{$_},
-    bounded => 1, font => { encoding => 'TS1' }) } keys %textcomp_various;
-
-#======================================================================
-RawTeX('\DeclareFontEncoding{TS1}{}{}');
-#======================================================================
```

Comment:
The software developer take on this is that we actually need a command that is almost exactly like `RequirePackage` but does not report loading the binding file, just treats it as a continuation of the caller. That way we could've kept the modularity, rather than move LaTeX.pool closer to 6000 lines of code.

Reply:
Why not simply add `RequirePackage('textcomp');` to `LaTeX.pool`?  So what if it reports? Nobody reads that anyway :>

Reply:
That was my first instinct! But then I saw what was done for `fixltx2e.sty` right above the code I added, and decided to "stay consistent with the current approach".

It appears that heuristic shouldn't be my default :> Happy to go either way, as long as we get the macros in.